startup/gui/outputs.py

```diff
@@ -195,11 +229,11 @@
 		)
 
 		GafferScene.Outputs.registerOutput(
-			"Batch/3Delight/" + label,
+			"Batch/3Delight/{}/{}".format( source, displayName ),
```

Comment:
Would be good to capitalise source here too I think.

Reply:
Fixed and squashed into https://github.com/GafferHQ/gaffer/pull/5463/commits/40f4be637a4480ef2d6b41f9171e1453a6fb9f74